README.md

```diff
@@ -52,8 +53,8 @@ class WordCountTest {
     private final WordCount app = new WordCount();
 
     @RegisterExtension
-    final TestTopology<Object, String> testTopology =
-        new TestTopology<>(this.app::getTopology, this.app.getKafkaProperties());
+    final TestTopologyExtension<Object, String> testTopology =
```

Comment:
somewhere, we'll probably also need examples of how the junit4 and the non-junit versions work.

Reply:
How about pointing to the tests instead?

Reply:
that should be fine. we just some pointer to where the info is.